sig/unit_test/convertibles.rbs

```diff
@@ -0,0 +1,154 @@
+module RBS
+  module UnitTest
+    module Convertibles
+      # An object that doesn't have methods excepts `__send__` and `__id__`
+      #
+      # Other methods are `undef`ed.
+      #
+      class BlankSlate < BasicObject
+        # Add Object methods to `self`
+        #
+        def __with_object_methods: (*Symbol methods) -> self
+      end
+
+      # An object with `#to_io` method
+      class ToIO < BlankSlate
+        @io: untyped
```

Comment:
Shouldn't these `untyped`s be given types? ditto for `initialize` and `to_xxx` for each of the `ToXXX` classes

Reply:
What is the type of IO here? I think it's not very clear and decided to keep them untyped.

Reply:
`@io` should be `::IO`; Same for `ToStr` and `String`, etc